conf/nginx/templates/nginx.conf.web.carbonio.admin.default.template

```diff
@@ -76,4 +76,17 @@ server
         # for custom error pages, internal use only
         internal;
     }
+
+    location ~/logout
+    {
+        add_header Set-Cookie "ZM_AUTH_TOKEN=; Path=/; Expires=Thu, 01-Jan-1970 00:00:00 GMT; Max-Age=0";
```

Comment:
- [ ] remove ZM_AUTH_TOKEN (clearing this will also logout any normal user session in browser which we dont want to do)